services/QuillLMS/.env-sample

```diff
@@ -73,7 +73,7 @@ STRIPE_SCHOOL_PLAN_PRICE_ID=price_2fe54c83fca05311a84a7ef5ab365e65
 ORTTO_API_KEY=
 
 # BIGQUERY
-BIGQUERY_CREDENTIALS=
+BIGQUERY_CREDENTIALS={}
```

Comment:
We want the e2e environment to copy .env-sample and load the variables. This yields a JSON parse error in the BigQuery initializer unless we default to a JSON parsable value